src/data/selector.ts

```diff
@@ -91,8 +91,11 @@ export interface ColorRGBSelector {
 
 export interface ColorTempSelector {
   color_temp: {
-    min_mireds?: number;
-    max_mireds?: number;
+    unit: "Kelvin" | "Mired";
```

Comment:
Would not use capitals here
```suggestion
    unit: "kelvin" | "mireds";
```